style.css

```diff
@@ -422,6 +501,238 @@ form textarea:focus {
   padding-left: 22%;
 }
 
+.primary,
 .social-media-icons {
   padding-top: 3em;
 }
+
+div {
+  display: block;
+}
+
+#section-contact p {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin-top: 1em;
+  margin-bottom: 3em;
+}
+
+#textarea {
+  background-image: url(./imges/Icon\ -\ Resizer\ Handle.svg);
+  background-repeat: no-repeat;
+  background-position: left 325px, 200px 385px, right 280px;
+  background-size: 130px;
+}
+
+#mobile-menu {
+  height: 85vh;
+  border-bottom-right-radius: 2em;
+  border-bottom-left-radius: 2em;
+  margin-bottom: 2em;
+  mix-blend-mode: normal;
+  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+}
+
+#mobile-menu > ul {
+  clear: right;
+  padding-left: 0;
+}
+
+.menu-link {
+  display: block;
+  text-decoration: none;
+  margin: 1em 1em;
+  color: #3a4a42;
+  font-size: x-large;
+  font-weight: 700;
+}
+
+.hidden {
+  display: none;
+}
+
+.social-media-icons li:hover {
+  transform: scale(1.3);
+}
+
+@media (min-width: 768px) {
+  /* Standard CSS rules */
+  html {
+    box-sizing: border-box;
+  }
+
+  header > a,
+  header > img {
+    display: none;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  #headline {
+    background-image: url("./imges/Desktop/Header-llustration.png");
+    background-size: 106% 16%;
+  }
+
+  .my-placeholder {
+    grid-template-columns: repeat(3, 1fr);
+    height: 372px;
+    width: 229px;
+  }
+
+  nav {
+    margin-top: -1em;
+  }
+
+  .middle {
+    grid-template-columns: repeat(3, 1fr);
+  }
```

Comment:
- [ ] As required in the project requirements you should make sure your design stick as much as possible to the Figma design. Your card still has some issues to fix like images, languages,...

Your design

<img width="675" alt="Screen Shot 2022-11-12 at 00 06 11" src="https://user-images.githubusercontent.com/74501523/201438872-7e515651-2e97-42eb-90c6-2dee6be04650.png">

Figma design

<img width="627" alt="Screen Shot 2022-11-12 at 00 08 02" src="https://user-images.githubusercontent.com/74501523/201438934-201c51c2-3b4b-46e8-9944-cbe4ad83e1ff.png">